Formula/srt.rb

```diff
@@ -0,0 +1,25 @@
+class Srt < Formula
+  desc "Secure Reliable Transport"
+  homepage "http://srtalliance.org"
+  url "https://github.com/Haivision/srt/archive/v1.2.0.tar.gz"
+  sha256 "ca22506cfafa18fb1f9c6dd5ceebe5e56c1b205105b2dd3736701aeea3f4145a"
+  head "https://github.com/Haivision/srt.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl"
+
+  def install
+    openssl = Formula["openssl"].opt_prefix
+
+    # SRT won't work appropriately with openssl which MacOS provides.
+    system "./configure", "--prefix=#{prefix}",
+                          "--with-openssl-includedir=#{openssl}/include",
+                          "--with-openssl-libdir=#{openssl}/lib"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/stransmit file:///dev/null file://con/ | cat -"
```

Comment:
I'd just like to understand what this does… and whether we could check its output

Reply:
For testing, I made tricky command because SRT has no test program. `stransmit` is a simple stream converting program. It parses two given URLs like `stransmit <source> <target>`

`file://con/` is reserved stdin for source or stdout for target so the command means acquiring a stream from null and sending to stdout.

Actually, the command in this patch just parses only URLs because I’d avoid using random port for testing. 

Generally, if there’s UDP stream, we can convert it to SRT stream by the below command. 
`stransmit udp://<host>:<port> srt://:<port>`

Reply:
@ilovezfs an opinion on that test?

Reply:
@fxcoudert @ilovezfs any advice for this patch?

Reply:
@justinjoy if the concern is about how to select a port,
```
    require "socket"

    server = TCPServer.new(0)
    port = server.addr[1]
    server.close
```
and then use `port`

Reply:
Actually, choosing random port is not a problem. Without a proper SRT client or server, `stransmit` will be blocked. That's my concern.